contrib/awslambda/python/src/python/pants/contrib/awslambda/python/examples/hello_handler.py

```diff
@@ -0,0 +1,12 @@
+# coding=utf-8
+# Copyright 2018 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+import pycountry
+
+from pants.contrib.awslambda.python.examples.hello_lib import say_hello
+
+
```

Comment:
Conventionally these would go in `examples` or `testprojects` folders at `contrib/awslambda/python/{examples,testprojects}`